app/models/services_metadata.rb

```diff
@@ -0,0 +1,44 @@
+require 'json'
+class ServicesMetadata
```

Comment:
If ServiceMetadata is never going to be instantiated then it makes more sense for it to be provided as a `Module`.

Reply:
Alternatively, you could create an object like `ServicesMetadata.new` which provides all the data and then provide a `to_json` method on it. This gives us some flexibility for presentation of the data when we need to.